File: Linguagem C++/crescente.cpp
#include <iostream>

using namespace std;

int main()
{
    int n1,n2;

    cout << "Digite dois numeros: \n";
    cin >> n1;
    cin >> n2;

        while (n1 != n2) {
            if (n1 < n2) {
                cout << "crescente \n";
            }
            else {
                cout << "decrescente \n";
            }
            cout << "Digite dois numeros: \n";
            cin >> n1;
            cin >> n2;
        }


    return 0;
}

File: Linguagem Python/EXs - Estruturas Repetitivas PY/crescente.py
print("Digite dois numeros: ")
num1 = int(input())
num2 = int(input())
if num1 != num2:
    while (num1 != num2):
        if num1 < num2:
            print("Crescente!")
        else:
            print("Decrescente!")
        print("Digite outros dois numeros: ")
        num1 = int(input())
        num2 = int(input())
